keyboards/pimoroni/keybow2040/info.json

```diff
@@ -0,0 +1,116 @@
+{
+    "manufacturer": "Pimoroni",
+    "keyboard_name": "Keybow 2040",
+    "url": "https://shop.pimoroni.com/products/keybow-2040",
+    "maintainer": "fanf2",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "features": {
+        "console": true
```

Comment:
```suggestion
        "console": true,
```

Reply:
Thanks for spotting those mistakes! New revision incoming!